data/wikipron/src/README.md

```diff
@@ -2,13 +2,15 @@
 ====================
 
 [`scrape.py`](scrape.py) calls WikiPron's scraping functions on all Wiktionary
-languages with over 100 entries. [`generate_summary.py`](generate_summary.py)
-generates a [README](../README.md) and a [TSV](../languages_summary.tsv) with
-selected information regarding the contents of the TSVs [`scrape.py`](scrape.py)
-generated and the configuration settings that were passed to scrape.
-[`postprocess`](postprocess) sorts and removes entries in each TSV if they have
-the same graphemic form and phonetic/phonemic form as a previous entry. In
-addition it splits TSVs containing multiple scripts (Arabic, Cyrillic, etc.)
+languages with over 100 entries. If a whitelist entry is present for language,
+will generate an extra 'filtered' file only containing whitelist phones/phonemes.
```

Comment:
Not sure why 'filtered' is quoted here...the meaning is clear, and I don't think you're doing any sort of ironic distance ;)